example-acord/app/project/DamlPlugin.scala

```diff
@@ -0,0 +1,27 @@
+import sbt.{AutoPlugin, File}
```

Comment:
This is the plugin that does not work too well inside sbt ?

Reply:
It works almost as I intended. Which does not work as I wanted is the incremental compilation. SBT has the facility to handle additional sources in an incremental fashion. Probably I configured it wrong.

The goal was when someone starts something with SBT it automatically would do the codegen and compilation of those files.

It seems like a good idea, but we can ditch this. I kept the plugin for now in case we have time to work out its issues.